normtag_BRIL.json

```diff
@@ -1717,6 +1790,7 @@
 ["pcc16PaperV5", {"274422": [[1450, 1644]]}],
 ["hfoc16PaperV5", {"274422": [[1645, 1859]]}],
 ["pcc16PaperV5", {"274422": [[1860, 2208]]}],
+["hfoc16v3b", {"274422": [[2209, 2211]]}],
```

Comment:
Why not use hfoc16PaperV5 here and in similar new HFOC allotments?

Reply:
@capalmer85 not covered by the updated https://github.com/CMS-LUMI-POG/Normtags/blob/a4d2401e1717203466a7dd8ed1727ecd52d75712/normtag_hfoc.json#L869